### PR TITLE
Add CalendarButton control.

### PR DESCRIPTION
**Pull request by StefanNenchev**

### CODE REVIEW

File: Controls/Input/Input.UWP/Themes/Calendar.xaml [hunk 1]
```diff
@@ -264,7 +263,41 @@
...
+    </Style>
 
+    <Style TargetType="localCalendar:CalendarFooterControl" x:Key="CalendarFooterControlStyle">
+        <Setter Property="FooterControlButtonStyle">
+            <Setter.Value>
+                <Style BasedOn="{StaticResource CalendarFooterControlButtonStyle}" TargetType="Button"/>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Height" Value="48"/>
+        <Setter Property="IsTabStop" Value="False" />
```
Comment (APopatanasov) [inline Controls/Input/Input.UWP/Themes/Calendar.xaml]: Why tap navigation is stopped?

File: Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs [hunk 2]
```diff
@@ -3652,11 +3697,16 @@ private Size CalculateAvailableCalendarViewSize(Size availableSize)
             availableSize.Width -= this.BorderThickness.Left + this.BorderThickness.Right + this.Padding.Left + this.Padding.Right;
             availableSize.Height -= this.BorderThickness.Top + this.BorderThickness.Bottom + this.Padding.Top + this.Padding.Bottom;
 
-            if (this.navigationPanel != null)
+            if (this.navigationPanel != null && this.HeaderVisibility == Visibility.Visible)
             {
                 availableSize.Height -= this.navigationPanel.ActualHeight;
             }
 
+            if (this.footerPanel != null && this.FooterVisibility == Visibility.Visible)
```
Comment (APopatanasov) [inline Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs]: When the visibility is changed dynamically are the measure and arrange phase triggered?

File: Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs [hunk 3]
```diff
@@ -297,6 +297,12 @@ public partial class RadCalendar : RadControl, IView, ICultureAware, ICollection
         public static readonly DependencyProperty HeaderVisibilityProperty =
             DependencyProperty.Register(nameof(HeaderVisibility), typeof(Visibility), typeof(RadCalendar), new PropertyMetadata(Visibility.Visible));
 
+        /// <summary>
+        /// Identifies the <c cref="FooterVisibility"/> dependency property.
```
Comment (APopatanasov) [inline Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs]: FooterVisibilityProperty

File: Controls/Input/Input.UWP/Calendar/View/CalendarFooterControl.cs [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
+
+namespace Telerik.UI.Xaml.Controls.Input.Calendar
+{
+    public class CalendarFooterControl : RadControl
+    {
+        /// <summary>
+        /// Identifies the <c cref=FooterControlButtonStyle"/> dependency property.
```
Comment (APopatanasov) [inline Controls/Input/Input.UWP/Calendar/View/CalendarFooterControl.cs]: FooterControlButtonStyleProperty

File: Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs (hunk 2 above)
Comment (StefanNenchev) [reply Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs]: yep

File: Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs (hunk 3 above)
Comment (StefanNenchev) [reply Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs]: you sure?

File: Controls/Input/Input.UWP/Themes/Calendar.xaml (hunk 1 above)
Comment (StefanNenchev) [reply Controls/Input/Input.UWP/Themes/Calendar.xaml]: Same as navigation control - the tab will be redirected to the button within the FooterControl instead.

**Pull request merged**